_harp/css/pages/landing/_landing-page.scss

```diff
@@ -270,6 +270,30 @@
 	}
 }
 
+.programmes-info{
+	height: 10vh;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	a {
+		margin: 2%;
```

Comment:
just noticing the css indentation in this file is really bad. Can we keep it consistent at 2 spaces?